index.html

```diff
@@ -674,26 +709,59 @@
       </p>
       <section>
         <h2>
-          <a>PushSubscriptionOptions</a> dictionary
+          <a>PushSubscriptionOptions</a> Interface and Dictionary
         </h2>
         <p>
-          A <a>PushSubscriptionOptions</a> object represents additional options associated with a
+          A <a>PushSubscriptionOptionsDict</a> represents additional options associated with a
           <a>push subscription</a>. The <a>user agent</a> MAY consider these options when
           requesting <a>express permission</a> from the user. When an option is considered, the
           <a>user agent</a> SHOULD enforce it on incoming <a>push messages</a>.
         </p>
-        <dl title="dictionary PushSubscriptionOptions" class="idl">
+        <p>
+          Once set, options for a <a>push subscription</a> cannot change. A pre-existing <a>push
+          subscription</a> can be <a href=
+          "#widl-PushSubscription-unsubscribe-Promise-boolean">unsubscribed</a> to create a <a>push
+          subscription</a> with new options.
+        </p>
+        <dl title="dictionary PushSubscriptionOptionsDict" class="idl" data-merge=
```

Comment:
PushSubscriptionInit? PushSubscribeOptions? I'd love to find a way to get rid of the *Dict suffix.